ls1intum/Apollon_standalone
# `Development`: Update dependencies

### Motivation and Context
This PR updates the dependencies of Apollon Standalone.

### Steps for Testing
1) Go to [test server](https://test1.apollon.ase.in.tum.de/)
2) Check that there are no issues with the application (styling and functionality)
